tango/step.py

```diff
@@ -49,30 +55,31 @@ def get_args(tp):  # type: ignore
 T = TypeVar("T")
 
 
+_random_for_step_names = random.Random()
+
+
 class Step(Registrable, Generic[T]):
     """
-    This class defines one step in your experiment. To write your own step, just derive from this class
+    This class defines one step in your experiment. To write your own step, derive from this class
     and overwrite the :meth:`run()` method. The :meth:`run()` method must have parameters with type hints.
 
     ``Step.__init__()`` takes all the arguments we want to run the step with. They get passed
     to :meth:`run()` (almost) as they are. If the arguments are other instances of ``Step``, those
     will be replaced with the step's results before calling :meth:`run()`. Further, there are four special
```

Comment:
This is not quite true either.

Reply:
Why not?

Reply:
I think when I first read this I interpreted it as "if the type hint of the argument is `Step` ...", but after reading it again I think it's clear.